phoenicis-configuration/pom.xml

```diff
@@ -51,7 +51,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>18.0</version>
+            <version>22.0-rc1-android</version>
```

Comment:
Release Candidates are also bad. In addition this seems to be an android version?